code/script.js

```diff
@@ -36,12 +41,103 @@ const showMessage = (message, sender) => {
 // Starts here
 const greetUser = () => {
   // here we call the function showMessage, that we declared earlier with the argument "Hello there, What's your name?" for message, and the argument "bot" for sender
-  showMessage("Hello there, What's your name?", 'bot')
+  showMessage("Konnichiwa Globetrotter! I'm an AI Travel Bot, what is your name?", 'bot')
   // Just to check it out, change 'bot' to 'user' here 👆
 }
 
 // Set up your eventlisteners here
 
+
+// Listen for the form submission event
+nameForm.addEventListener('submit', (event) => {
+  event.preventDefault(); // Prevent the form from submitting
+
+  if(userName == null){ // Check if userName is not already set
+  userName = nameInput.value; // Set userName from the input field
+  showMessage(userName, 'user'); // Display user name
+  nameInput.value = ''; // Clear the input field
+
+  // Display a welcome message from the bot after a short delay
+  setTimeout(() => {
+    const botMessage = `Hello ${userName}! Let's see the world!`;
+    showMessage(botMessage, 'bot');
+  }, 500);
+
+  // User options after a delay
+  setTimeout(() => {
+    const botMessage = `This is what I can do for you: 
+<p>1 - Book a flight</p>
```

Comment:
For example you could move this "p's" so they are underneath botMessage, I hope you understand what I'm trying to say 🙈